definitions/region/tiam-ecn_1.2.yaml

```
@@ -0,0 +1,37 @@
+- TIAM-ECN 1.2:
+  - TIAM-ECN 1.2|ACE
```

You could use the renaming step to provide more user-friendly names…
```suggestion
  - TIAM-ECN 1.2|<some more descriptive name>
```